### ISSUE TITLE
error callback

### ISSUE THREAD

**Issue by PalmerAL** [nt1m/InfoCards.js#7]
On errors (such as entering a really long text string), the api just returns a blank file. Infocards should have some kind of error callback to handle this.

**Comment by nt1m** [comment]
A really long string isn't really an error, it's just a query that returns no results, kinda like how google has no results for long strings that have no meanings.

**Comment by nt1m** [comment]
I could add an error callback if the request to duckduckgo is failing though.

**Comment by PalmerAL** [comment]
No, these are two separate things. No results returns a file with empty arrays. Something really long (like a paragraph or two) just returns a completely blank file with nothing in it.

**Comment by nt1m** [comment]
@PalmerAL I'd call onEmpty in this case.
